dash/_callback_context.py

```diff
@@ -9,9 +9,10 @@ def has_context(func):
     def assert_context(*args, **kwargs):
         if not flask.has_request_context():
             raise exceptions.MissingCallbackContextException(
-                'dash.callback.{} is only available from a callback!'.format(
-                    getattr(func, '__name__')
-                )
+                (
+                    'dash.callback_context.{} '
+                    'is only available from a callback!'
+                ).format(getattr(func, '__name__'))
```

Comment:
Why change? I find that less readable. The parens are unnecessary.

Reply:
The previous one no longer fit in 80 characters when I fixed `dash.callback` -> `dash.callback_context`.

Also, not sure if this is a pylint version issue or what, but the previous way it was written I was getting mutually exclusive errors about indentation: CI required it to be one way, my local pylint required something else. This way they both agree is acceptable.

Reply:
oh no sorry, the pylint issue was somewhere else. This was just >80 chars.